resources/pvc.py

```diff
@@ -102,7 +102,7 @@ def get_all_pvcs():
     return out
 
 
-def create_multiple_pvc(number_of_pvc=1, pvc_data=CSI_PVC_DICT.copy()):
+def create_multiple_pvc(number_of_pvc=1, pvc_data=CSI_PVC_DICT.deepcopy()):
```

Comment:
I think dict does not have deepcopy attribute

Reply:
right, stupid mistake on my side

Reply:
fixed